sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -1448,6 +1466,64 @@ void captureSessionToken(RxDocumentServiceRequest request, RxDocumentServiceResp
                 .map(resp -> toResourceResponse(resp, Document.class));} );
     }
 
+    @Override
+    public Mono<ResourceResponse<Document>> patchDocument(String documentLink,
+                                                          List<PatchOperation<?>> patchOperations,
+                                                          RequestOptions options) {
+        DocumentClientRetryPolicy requestRetryPolicy = this.resetSessionTokenRetryPolicy.getRequestPolicy();
+        if (options == null || options.getPartitionKey() == null) {
+            String collectionLink = Utils.getCollectionName(documentLink);
+            requestRetryPolicy = new PartitionKeyMismatchRetryPolicy(collectionCache, requestRetryPolicy, collectionLink, options);
+        }
+
+        DocumentClientRetryPolicy finalRetryPolicyInstance = requestRetryPolicy;
+        return ObservableHelper.inlineIfPossibleAsObs(() -> patchDocumentInternal(documentLink, patchOperations, options, finalRetryPolicyInstance), requestRetryPolicy);
+    }
+
+    private Mono<ResourceResponse<Document>> patchDocumentInternal(String documentLink,
+                                                                   List<PatchOperation<?>> patchOperations,
+                                                                   RequestOptions options,
+                                                                   DocumentClientRetryPolicy retryPolicyInstance) {
+        if (patchOperations == null) {
+            throw new IllegalArgumentException("patchOperations");
+        }
+
+        logger.debug("Running patch operations on document. patch Operations: [{}]", patchOperations);
+
+        final String path = Utils.joinPath(documentLink, null);
```

Comment:
Needed?

Reply:
This utility append "/" and trims whitespace, so seems fine.